nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -254,7 +254,7 @@ let
           # UDP listener for **QUIC+HTTP/3
           listen ${addr}:${toString port} http3 reuseport;
           # Advertise that HTTP/3 is available
-          add_header Alt-Svc 'h3=":443"';
+          add_header Alt-Svc 'h3-29=":443"; ma=86400';
```

Comment:
You should probably explain what is fixed by this and how.

Reply:
Also why? Firefox, Chrome and curl all work with the h3 ALPN. Also 86400 is 24h which is the default